manuscript/TableS1.jl

```diff
@@ -14,7 +14,6 @@ isdir(tablefiles()) || mkpath(tablefiles())
 
 ## 1. Loading the model files
 
-RandomForestClassifier = MLJ.@load RandomForestClassifier pkg=DecisionTree
 RandomForestRegressor = MLJ.@load RandomForestRegressor pkg=DecisionTree
 # concurrent cogScore reression from taxonomic profiles
 JLD2.@load "models/2023-02-15/regression_currentCogScores_00to06mo_onlytaxa.jld"
```

Comment:
Not that all of these are getting changed in #131 (though I have not yet gone through and saved the `save` ones.